forem/forem
# Remove Setup Banner (VerifySetupCompleted)

## What type of PR is this? (check all applicable)


## Description
This PR removes the `VerifySetupCompleted` banner from the site and removes the code associated with it from the codebase. 

## Related Tickets & Documents
N/A

## QA Instructions, Screenshots, Recordings
**To QA this PR, please click around the site and ensure that the setup banner isn't visible anywhere, especially on the Creator Settings page, the homepage, and in the Admin portal, and that the site works as expected.**

### Before:
<img width="1348" alt="Screen Shot 2022-01-05 at 1 50 19 PM" src="https://user-images.githubusercontent.com/32834804/148287479-a7ccb4ea-c146-4899-a057-e05f186db06b.png">

### After:
<img width="1360" alt="Screen Shot 2022-01-05 at 1 51 23 PM" src="https://user-images.githubusercontent.com/32834804/148287620-349527e2-4a54-4bf1-bd9b-39df4a269215.png">

### UI accessibility concerns?
N/A

## Added/updated tests?

- [x] No, and this is why: this PR removes a feature.

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
- [x] I'm not sure how best to communicate this change and need help: Should we communicate this to the broader team or to the community?
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

![Child sliding down a slide out of view](https://media.giphy.com/media/m9eG1qVjvN56H0MXt8/giphy.gif)
